frontend/src/modules/machinery/components/source/ConcordanceSearch.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+type Props = {|
+    projectName?: ?string,
+|};
+
+/**
+ * Show the concordance search results from Pontoon's memory.
+ */
+export default function ConcordanceSearch(props: Props) {
+    return (
+        <li>
+            <Localized
+                id='machinery-ConcordanceSearch--pontoon-homepage'
+                attrs={{ title: true }}
+            >
+                <a
```

Comment:
How is an anchor element useful here if its action is prevented?

Reply:
We just prevent propagation in order to prevent copying the content into the editor when clicking on header.

But I do think we should drop these links - no point in linking to Pontoon Homepage - especially in the light of this proposal: https://github.com/mozilla/pontoon/pull/1794#issuecomment-778156337.